samples/file-encryptor/enclave/CMakeLists.txt

```diff
@@ -29,6 +29,23 @@ target_include_directories(
   PRIVATE ${CMAKE_CURRENT_SOURCE_DIR} # Needed for #include "../shared.h"
           ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_SOURCE_DIR})
 
-target_link_libraries(
-  enclave openenclave::oeenclave openenclave::oecrypto${OE_CRYPTO_LIB}
-  openenclave::oelibcxx)
+if (${OE_CRYPTO_LIB} STREQUAL "openssl_symcrypt_fips")
+  add_custom_command(
+    TARGET enclave
+    COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/libsymcrypt.so.103
+            ${CMAKE_CURRENT_BINARY_DIR}/libsymcrypt.so.103)
+
+  target_link_libraries(
+    enclave
+    openenclave::oeenclave
+    openenclave::oesymcryptengine
+    openenclave::oecryptoopenssl
+    # Workaround: refer to the root of the build location so that we don't rely on the
+    # order of copy
+    ${CMAKE_BINARY_DIR}/libsymcrypt.so.103
```

Comment:
It would be better to avoid hard-coding the version number here.
The download steps can create the necessary symlinks or copies.